.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer self-update
+  - composer install --prefer-source --no-interaction
```

Comment:
> Testing the most recent/fresh version is recommended.

`--prefer-dist` will get exactly the same versions. The only difference is the download instead of a more long git clone.

For the cache, I was talking about Travis home cache. Please see this sample: https://github.com/Soullivaneuh/IsoCodesValidator/blob/master/.travis.yml#L36-L38